static/src/javascripts/projects/commercial/modules/dfp/non-refreshable-line-items.ts

```diff
@@ -0,0 +1,31 @@
+import { memoize } from 'lodash-es';
+import reportError from 'lib/report-error';
+
+export const fetchNonRefreshableLineItemIds = async (): Promise<
+	number[] | undefined
+> => {
```

Comment:
I think this is really a `Promise<number[]>`. Either you get an array of numbers, or it fails via `reportError`. Returning `undefined` is not that useful, we should reject the promise instead.

```suggestion
export const fetchNonRefreshableLineItemIds = async (): Promise<number[]> => {
```

Reply:
Updated this so that the function throws an error when fetching/parsing the array of numbers, which can then be caught and handled downstream.